### ISSUE TITLE
Reseting the camera?

### ISSUE THREAD

**Issue by marcoscaceres** [immersive-web/model-element#42]
As the model can be interactive, it's foreseeable that users might place/rotate a model in such a way that they are unsure which direction the model is facing. It might be nice to provide some way of resetting the camera to its initial position. 

This is a "nice to have" (i.e., probably not critical) because a developer could take a snapshot of the camera's position when the model loads. However, that's kinda annoying because then they need to keep their own accounting of the starting position of each model.

**Comment by cabanier** [comment]
I think this only makes sense if you allow controls (like in proposal #43).
Otherwise, shouldn't the model be completely under the author's control?

**Comment by marcoscaceres** [comment]
> Otherwise, shouldn't the model be completely under the author's control?

Yes, to a degree... it depends on how the rendering is presented, which is up to the user agent: the author might not have control over the placement of the initial camera (the user agent might decide what the best was to present the scene is, so it can show as much as possible of the whole scene).
